08_files_lab/01.py

```diff
@@ -1,6 +1,9 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+#Append file 1 into the end of file 2
+import sys
+
+(src, dst) = sys.argv[1:]
+
+with open(src, "r") as fin: #Open file for reading
+    with open(dst, "a+") as fout: #Open file for appending. If the file doesn't exist then create it
+        for line in fin:
+            fout.write(line) #Copy each line

```

Comment:
🍪 for taking from argv only what you need.
Better not to add that + sign after the "a" in the opening as this means the file is open for reading as well. Which we don't need.